pkg/controller/configmap/configmap_controller.go

```diff
@@ -221,13 +248,13 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 		return reconcile.Result{}, err
 	}
 	if !ncpNeedChange && !agentNeedChange {
-		// Check if NCP_IMAGE changes
-		ncpImageChanged, err := r.isNcpImageChanged()
+		// Check if NCP_IMAGE and nsx-ncp replicas change
```

Comment:
`Check if NCP_IMAGE and nsx-ncp replicas change` -> `Check if NCP_IMAGE or nsx-ncp replicas change` or just remove the comment

Reply:
Sure